app/models/story.rb

```diff
@@ -117,6 +120,14 @@ def tags=(ts)
     self.user_tags = ts.uniq.sort.map { |t| UserTag.new(name: t) }
   end
 
+  def transcript
```

Comment:
What is this story/audio_file transcript supporting?  Just a coincidental feature?